### DISCUSSION TITLE
DevTools: Trying to access array offset on value of type null

### DISCUSSION THREAD

**Discussion by lukepower** [Laravel-Backpack/community-forum#367] [Q&A (Help)] [answered]
Hi there,
just bought and wanted to use DevTools for a new model, but even with the default values I keep getting this error (along with a Server 500):

Trying to access array offset on value of type null

`{
    "class": "ErrorException",
    "message": "Trying to access array offset on value of type null",
    "code": 0,
    "file": "F:\\Programming\\new-remote-skygems\\vendor\\laravel-shift\\blueprint\\src\\Tracer.php:120",
    "trace": [
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Bootstrap\\HandleExceptions.php:266",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel-shift\\blueprint\\src\\Tracer.php:120",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Collections\\Traits\\EnumeratesValues.php:236",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel-shift\\blueprint\\src\\Tracer.php:121",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel-shift\\blueprint\\src\\Tracer.php:33",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel-shift\\blueprint\\src\\Commands\\TraceCommand.php:51",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\BoundMethod.php:36",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Util.php:41",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\BoundMethod.php:93",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\BoundMethod.php:37",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php:663",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Console\\Command.php:182",
        "F:\\Programming\\new-remote-skygems\\vendor\\symfony\\console\\Command\\Command.php:312",
        "F:\\Programming\\new-remote-skygems\\vendor\\laravel\\framework\\src\\Illuminate\\Console\\Command.php:152",`

Any idea what could cause this?

**Comment by jcastroa87** [comment]
Hello @lukepower 

Im sorry for super late answer, can you please make an update of this issue, still happen after composer update?

Can you share you version:
```
php artisan backpack:version
```

Cheers.

**Reply by lukepower** [reply, marked as answer]
I was able to fix it by updating all dependencies

**Reply by jcastroa87** [reply]
Glad to hear.

Cheers.
